setup.py

```diff
@@ -53,7 +53,10 @@
         "reinforcement learning "
     ),
     install_requires=["gym", "ai-safety-gridworlds", "numpy>=1.14.5"],
-    dependency_links=["https://github.com/jvmancuso/ai-safety-gridworlds/tarball/master#egg=ai-safety-gridworlds-1.2.2"],
+    dependency_links=[
+        "https://github.com/jvmancuso/gym/tarball/master#egg=gym-0.10.10",
```

Comment:
The version of gym from PyPI is 0.10.9 and was released 5 Nov, 2018, so any PRs merged to master since then aren't on PyPI.  openai/gym#492 was merged after that.  It seemed reasonable to point our `dependency_link` arg to their master branch since the package is now in maintenance mode, but setuptools wasn't able to find it and would fallback to the 0.10.9 version on PyPI.

I can try again to see if that'll work (cause there were other complicating factors at the time that I found later), but what I ended up doing was making a release on my fork and pointing our dependency_link to that.

Reply:
Sure, that makes sense. Pointing it to their master branch works for me, see https://github.com/david-lindner/safe-grid-gym/pull/14/commits/cfcf23b8f2eb3447f3567ace4653fffa807fcb7e Have a look if it works for you as well :smile: